vsBuild.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+cd de.variantsync.core
+mvn clean verify
+eLevel=$?
```

Comment:
What is eLevel? Please add a comment.

Reply:
eLevel= error level = returned error code by last executed command. `eCode` would be better, I guess

Reply:
Or maybe `errorCode`? :D The aftermaths of C still lives on.